cognizant-studio-academy/Inventory-Management
# Display menu according to the roleId

#### What's this PR do?
This PR contains Manage Inventories menu
#### Where should the reviewer start?
All Files in Client/ 
#### How should this be manually tested?
Step 1 : Login whose role is admin
Step 2: verify the Manage Inventories menu is displaying or not
#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/166152378
